samples/Grafana/README.md

```diff
@@ -0,0 +1,55 @@
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Flocalmachine)
+
+# Configure `dotnet monitor` in an AKS cluster for Managed Grafana
+
+`dotnet monitor` provides snapshots of .NET metrics in the Prometheus exposition format. [Prometheus](https://prometheus.io/docs/introduction/overview/) in turn collects metrics from targets by scraping metrics HTTP endpoints.
+
+This doc provides instructions on customizing metrics scraping for a Kubernetes cluster with the metrics addon in Azure Monitor and how to subsequently renders those metrics with Grafana.
+
+### Step 1: `dotnet monitor` configuration
+
+The following settings ensure that the [metrics endpoint is bound to an external address](https://github.com/dotnet/dotnet-monitor/blob/main/documentation/configuration/metrics-configuration.md#metrics-urls), and not the internal localhost. Today we also recommend setting a fairly low scraping interval for the metric count as follows.
+
+```yaml
+Metrics__Endpoints: http://+:52325
+Metrics__MetricCount: '1'
+```
+
+### Step 2: Include deployment annotations
+You will need to update cluster deployment to scrape certain pods and specify the port, path, and scheme through annotations for the pod.
+
+```yaml
+annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/path: '/metrics'
+    prometheus.io/port: '52325'
+    prometheus.io/scheme: 'http'
+```
+
+### Step 3: Apply ConfigMap
+
+You can download this [metrics settings config map file](https://github.com/Azure/prometheus-collector/blob/main/otelcollector/configmaps/ama-metrics-settings-configmap.yaml) and change the settings as appropriate. The `podannotationnamespaceregex` setting requires an update to ensure that it matches the namespace configured for your app (check your deployment). If your namespace is blank or undefined `podannotationnamespaceregex` will become 'default' as follows.
+
+```yaml
+podannotationnamespaceregex = "default"
+```
+
+Save your ConfigMap and apply/deploy to the kube-system namespace for your cluster as follows (only required once).
+
+```shell
+kubectl apply -f .\ama-metrics-settings-configmap.yaml -n kube-system
+```
+
+This configures the Prometheus agent to check the default namespace for active pods and use the annotations in the pod (step 2) to scrape for Prometheus data, the scraping will occur on an interval defined in the ConfigMap.
```

Comment:
Consider adding something like "It may take a few minutes for your changes to take effect". I noticed in my cluster it took a few minutes for metrics to start propagating even with a small interval.